showcase-app/plugin/identityPlugin.js

```diff
@@ -17,6 +24,8 @@ export default class IdentityWidget {
    */
 
   _initializeConnection = () => {
```

Comment:
Documentation missing.

Reply:
found a way to pin the extension id, so it should work for every user without the need to change the hardcoded extension id